src/main/ruby/jruby/rack/helpers.rb

```diff
@@ -134,7 +134,7 @@ def resolve_constant(camel_cased_name, context = Object)
             require underscore(camel_cased_name)
             retry
           rescue LoadError => le
-            e.message << " (#{le.message})"
+            e.message = "#{e.message} (#{le.message})"
```

Comment:
Some older Rails/Jruby combinations complain that the `NameError.message` is a frozen string literal when hitting this code; better to avoid assuming I suppose?